client/src/pages/trainer/course/[id]/tableList.tsx

```diff
@@ -0,0 +1,79 @@
+import { Fragment } from 'react';
+import type { FC } from 'react';
+import Button from '@/src/shared/components/Button';
+import Table from '@/src/shared/components/Table';
+import Navbar from '@/src/shared/components/Navbar';
+import { dropdownItems, navItems } from '@/src/pages/demo/layouts/navbar';
+
```

Comment:
![image](https://user-images.githubusercontent.com/73863685/225811588-dee4839f-a4f6-44d9-b9af-60a04bf5a997.png)
- since this is reusable. move this to component page lang.